libyui-ncurses/pkgconfig/libyui-ncurses.pc.in

```diff
@@ -0,0 +1,23 @@
+#
+# Pkg-Config file for libyui-ncurses
+#
+# Generated by CMake from libyui-ncurses.pc.in
+#
+
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${exec_prefix}/include/yui/ncurses
+plugindir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@/yui
+
+soversion_major=@SONAME_MAJOR@
+soversion_minor=@SONAME_MINOR@
+soversion_patch=@SONAME_PATCH@
+soversion=@SONAME@
+
```

Comment:
as for plugindir i think soversion should not be needed since libyui wins to load plugins and extensions.

Reply:
I am not sure; it might be possible that this turns out to be useful in the future in some CMake magic on the consumer side (in a plug-in extension using this). Probably it's not needed, but I don't think it will hurt. Plus, it was there in the old .pc files.